packages/ui/src/kit/VInputGroup/VInputGroup.theme-default.scss

```diff
@@ -0,0 +1,31 @@
+.v-input-group.theme-default {
+  display: flex;
+  justify-content: stretch;
+  width: 100%;
+}
+
+.v-input-group.theme-default .field-atom:first-child {
+  border-right: 1px solid var(--endpass-ui-color-grey-2);
+}
+.v-input-group.theme-default .field-atom:first-child .input-atom,
```

Comment:
Add `flex` property for each flex container's children components. We should handle `flex-growth` and `flex-shrink` properties behavior

Reply:
This is sensitive issue. By default, all flex descendants have `flex-grow: 1`, all the same, and have the same width that allows them to fit container properly. We cannot handle sizes of elements we nothing know about, in abstract component like that. Inside we may have selects, inputs, and many more in future, so we cannot add explicit `grow` or `shrink` values. And of course, we may suppress explicit values in favor of default ones.

But, in component which will use `v-input-group`, we will be able to tune sizes of group descendants, play with sizes, with `flex-grow` and `flex-shrink`. Because we will know, what are we dealing with. Just like in my `VUrlWithProtocol` component in `vault`.